packages/esm-patient-chart-app/src/visit/hooks/useDeleteVisit.test.tsx

```diff
@@ -56,14 +58,13 @@ describe('useDeleteVisit', () => {
   beforeEach(() => {
     jest.clearAllMocks();
```

Comment:
Unrelated to this diff, but we're already clearing all mocks in the top-level jest config, so this call and the one on line 141 are redundant.